src/pages/workspace/rules/ExpenseReportRulesSection.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import usePolicy from '@hooks/usePolicy';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as CurrencyUtils from '@libs/CurrencyUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import * as PolicyActions from '@userActions/Policy/Policy';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+type ExpenseReportRulesSectionProps = {
+    policyID: string;
+};
+
+function ExpenseReportRulesSection({policyID}: ExpenseReportRulesSectionProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const policy = usePolicy(policyID);
+
+    // Auto-approvals and self-approvals are unavailable due to the policy workflows settings
+    const workflowApprovalsUnavailable = policy?.approvalMode !== CONST.POLICY.APPROVAL_MODE.BASIC || !!policy?.errorFields?.approvalMode;
```

Comment:
Currently, in our App, I see we have three approval mode
- OPTIONAL: Turn off the approval option in workflow section
- BASIC: Turn on approval option in workflow section
- ADVANCED: Turn on the approval option in workflow section and there are many approval workflows or many approvers

and I think we only need to lock Auto-approvals and self-approvals when approval mode is OPTIONAL

@marcaaron Please help to verify my thought

```suggestion
    const workflowApprovalsUnavailable = policy?.approvalMode === CONST.POLICY.APPROVAL_MODE.OPTIONAL || !!policy?.errorFields?.approvalMode;
```

Reply:
Waiting for @marcaaron to confirm

Reply:
Yes, so, on OldDot we let you control this at anytime. But in NewDot we should use whatever logic we already use for the "Approvals" toggle. Coming from the "Advanced approvals" design doc it states:

>When Add approvals are disabled, that’s the equivalent of OldDot’s Submit and Close. Enabled is the same as Submit and Approve. Both of these already exist as part of [Simplified Collect](https://docs.google.com/document/u/0/d/1Qv6V4nRqIbZfzQML_saCbrAEqQVIBsiQ3HE1UO7RCxQ/edit), and by convention, the default approver when enabling Approvals is the policy owner.

Submit and Close = `OPTIONAL` ("Add approvals" disabled)
Submit and Approve = `BASIC` ("Add approvals" enabled)
Advanced Approval = `ADVANCED` ("Add approvals" would also be enabled in this case I'm guessing)

So I think the existing logic is wrong and it really should be checking for a value that is not BASIC or ADVANCED (which is more or less the suggestion made by @DylanDylann).